Automattic/wp-calypso
# Private Site: Migrate Image to Webpack

#### Changes proposed in this Pull Request

This migrates the one image in `/private` to Webpack. For details on the reasoning behind this, see #36850.

#### Testing instructions

There should only be one use of this image: https://github.com/Automattic/wp-calypso/search?q=%22private.svg%22&unscoped_q=%22private.svg%22

You can find it by visiting `/log-in/private-site`. Verify that the image is still there and no different. 

<img width="649" alt="Screenshot 2019-11-09 at 13 01 57" src="https://user-images.githubusercontent.com/43215253/68528974-5da9bb80-02f1-11ea-9af3-0c8e03786ea7.png">

cc @jsnajdr, @scruffian
